decidim-admin/app/jobs/decidim/admin/verify_user_group_from_csv_job.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Admin
+    # Custom ApplicationJob scoped to the admin panel.
+    #
+    class VerifyUserGroupFromCsvJob < ApplicationJob
+      queue_as :default
+
+      def perform(email, verifier, organization)
+        @organization = organization
+        @email = email
```

Comment:
Should we downcase and strip the email in case they have written it wrong at the CSV?

Reply:
Ok!